### DISCUSSION TITLE
Compound objects "multiple" icon

### DISCUSSION THREAD

**Discussion by SergioAlias** [CollectionBuilder/collectionbuilder.github.io#140] [Q&A] [answered]
**Reference webpage and code:**

- https://sascards.github.io
- https://github.com/SASCards/SASCards.github.io

--

Hello and thank you for your work.

As stated in the [documentation](https://collectionbuilder.github.io/cb-docs/docs/metadata/compound-objects/#metadata-conventions):

> In CollectionBuilder-GH, compound objects will be represented by icons in visualizations.

I'm creating a business card collection that uses "multiple" items to store 2 images (front and back of the business card). I would like to set the front image as the default in visualizations instead of the current icon showed:

**Current behaviour**
![imagen](https://github.com/user-attachments/assets/632bcb21-87b3-42ad-b599-2b339f5090b1)

**Wanted behaviour**

Instead of that icon, use the first image of the "multiple" item, or any image specified by user in metadata.

**What I already tried**

- Add the filename to the parent item `filename` field

**Comment by evanwill** [comment, marked as answer]
@SergioAlias it is technically possible to customize CB-GH to implement what you are asking, but it will involve editing parts of every page layout. Honestly, the easiest approach would be to use collectionbuilder-csv instead, which supports what you want by default. 

Migrating into to CB-CSV from -GH is pretty straightforward. All the configuration and metadata that you have in your current project are transferable into a CB-CSV project. The main difference is that CB-CSV adds the columns "object_location", "image_thumb", and "image_small" to the metadata, instead of using the "filename" field that CB-GH depends on. 
CB-GH does not use thumbnails, instead using the full sized image for as listed in your "filename" field. CB-CSV supports adding thumbnail and small images for any object type--however, you could just use the same image like CB-GH does. 
To migrate You can take your "filename" field and use it to fill in the "object_location" field by adding `/objects/` in front of the filename value, e.g. `/objects/demo001.jpg` or `/objects/demo002.pdf`. For the image items, you would then fill in  "image_small" and "image_thumb" fields with exactly the same value. For your parent "multiple" item, you would fill in the  "image_small" and "image_thumb" fields with which ever image you want to use as the representative image. 

Building a [CB-CSV project on github pages](https://collectionbuilder.github.io/cb-docs/docs/deploy/actions/) is pretty straightforward now too (just barely more difficult than standard GH Pages!).

**Comment by SergioAlias** [comment]
Thank you so much for your quick response @evanwill ! It looks like I completely misunderstood what a CB template is, so I was discarding the one without GH in the name before looking more in-depth. I'll migrate my project. Thanks!
